ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -131,19 +132,19 @@ static EngineExecutor create(
 
   ExecuteResult execute(final KsqlPlan plan) {
     final Optional<QueryPlan> queryPlan = plan.getQueryPlan();
-
+    final String maskedStatement = QueryMask.getMaskedStatement(plan.getStatementText());
```

Comment:
Humm... I just noticed that this change is actually redundant. Assuming we've propagated the masking through properly everywhere else, then this statement text should already be masked.

Reply:
Thanks! Let me look into that

Reply:
Planning to add the types in the master and keep these branches as clean as possible. Would that make sense?

Reply:
Makes sense. I don't mind the redundancy on these older branches as long as it's cleaned up in master, but if you want to remove the redundancy I am quite confident that it is safe (a second pair of eyes never hurts though :) )